geowebcache/core/src/main/java/org/geowebcache/layer/TileLayerDispatcher.java

```diff
@@ -302,6 +309,12 @@ public void afterPropertiesSet() throws Exception {
         this.configs =
                 GeoWebCacheExtensions.configurations(
                         TileLayerConfiguration.class, applicationContext);
+
+        XMLConfiguration config = applicationContext.getBean(XMLConfiguration.class);
```

Comment:
I think this should be BaseConfiguration.class so it can work with alternate implementations like Stratus.